Sources/Basics/Archiver+Zip.swift

```diff
@@ -14,7 +14,7 @@ import TSCBasic
 import Dispatch
 
 /// An `Archiver` that handles ZIP archives using the command-line `zip` and `unzip` tools.
-public struct ZipArchiver: Archiver, Cancellable {
+public final class ZipArchiver: Archiver, Cancellable {
```

Comment:
ideally this remains a struct

Reply:
Addressed in the base `TarArchiver` PR, both `FileSystem` and `Cancellator` are reference types, and `Cancellator` in particular holds mutable state, which transitively makes `ZipArchiver` stateful.